tc/sgx/trusted_worker_manager/enclave/enclave.edl

```diff
@@ -17,12 +17,6 @@ enclave {
     from "sgx_tstdc.edl" import *;
     from "sgx_tsgxssl.edl" import *;
     from "base.edl" import *;
-    from "signup.edl" import *;
```

Comment:
signup.edl is  common for all encalves types

Reply:
Included singup.edl inside enclave.edl file.